Hmwk_#9_MONGODB/sqlite_to_mongo.py

```diff
@@ -0,0 +1,67 @@
+import datetime
+import sqlalchemy
+import json
+from pymongo import MongoClient
+
+
+source_db_path = 'sqlite:///C:\\sqlite\\sqlite_DBs\\orest_database.db' #use local path to SQLite DB file
+
+#connect to SQLite source DB and retrieve data for migration to MongoDB
+def getSourceData(source_db_path):
+
+    engine = sqlalchemy.create_engine(source_db_path)
+    sqlite_source = engine.connect() #connecto to source DB
+
+    projects = sqlite_source.execute('SELECT * FROM Project')  #query data from tables
+    tasks = sqlite_source.execute('SELECT * FROM Tasks')
+
+    projects_data = json.dumps([(dict(row.items())) for row in projects]) #convert o JSON for migration
+    tasks_data = json.dumps([(dict(row.items())) for row in tasks]) #convert o JSON for migration
```

Comment:
It can be slightly simplified: 
```
projects_data = json.dumps([dict(row) for row in projects])
tasks_data = json.dumps([dict(row) for row in tasks])
```